test/Test/Oscoin/Crypto/Address/Gen.hs

```diff
@@ -0,0 +1,22 @@
+module Test.Oscoin.Crypto.Address.Gen
+    ( address
+    ) where
+
+import           Oscoin.Crypto.Address
+import           Oscoin.Crypto.Hash (HasHashing)
+import           Oscoin.Crypto.PubKey (HasDigitalSignature, generateKeyPair)
+import           Oscoin.Test.Crypto.PubKey.Arbitrary
+import           Test.Oscoin.Crypto.Address (genAddress)
+
+import           Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+import qualified System.Random.SplitMix as SplitMix
+
+address :: MonadGen m => (HasHashing c, HasDigitalSignature c) => m (Address c)
```

Comment:
```suggestion
address :: (MonadGen m, HasHashing c, HasDigitalSignature c) => m (Address c)
```

Reply:
Shouldn’t we use `genAddress` by convention? Shouldn’t `Test.Oscoin.Cyrpto.Address.genAddress` also live in this module?

Reply:
Looking at some of the other `*.Gen` modules, I think you're right - there's only one module that doesn't do this - `Config.Gen` :thinking:

Reply:
0b172de